languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/french_tags.txt

```diff
@@ -32,55 +34,55 @@ N m p
 N m s
 N m sp
 P
-R 2 s
-R 3 s
-R dem e s
```

Comment:
@OswaldLuc 
Before anything else, thank you for the great work.
I would like to use your work, but, I believe that, `just like languagetool-org`, the change in some tags is cause of concern. You have fixed some rules, but other rules may have changed results that are very hard to track.
For example, the changes in tag `R refl` or `S` may be easy to correct, but in the re-categorization of `R e s` if might be trickier.
Is it possible to make the parser map the tags to the old standard?